src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -695,8 +707,76 @@ public NodeViewModel(WorkspaceViewModel workspaceViewModel, NodeModel logic)
             ++NoteViewModel.StaticZIndex;
 
             if(workspaceViewModel.InCanvasSearchViewModel.TryGetNodeIcon(this, out ImageSource imgSource)) ImageSource = imgSource;
+
+            // The Node displays a count of dismissed messages, listening to that collection in the node's ErrorBubble
+            ErrorBubble.DismissedMessages.CollectionChanged += DismissedNodeWarnings_CollectionChanged;
```

Comment:
@QilongTang @OliverEGreen @SHKnudsen can we unsubscribe these handlers in the NodeViewModel.Dispose please?